frontend/lib/ui/footer.tsx

```diff
@@ -2,18 +2,20 @@ import React from "react";
 import { PrivacyPolicyLink, TermsOfUseLink } from "./privacy-info-modal";
 import JustfixRoutes from "../justfix-routes";
 import { ROUTE_PREFIX } from "../util/route-util";
+import { Trans } from "@lingui/macro";
+import { LegalDisclaimer } from "./legal-disclaimer";
 
 const CreditForLHI = (props: { pathname?: string }) =>
   /* Include credit for LHI only on ehp routes */
-  props.pathname &&
-  props.pathname.startsWith(JustfixRoutes.locale.ehp[ROUTE_PREFIX]) ? (
+  props.pathname?.startsWith(JustfixRoutes.locale.ehp[ROUTE_PREFIX]) ||
+  props.pathname?.startsWith(JustfixRoutes.locale.hp[ROUTE_PREFIX]) ? (
```

Comment:
This isn't related to i18n but if/when we re-enable the regular HP action flow, we'll want this disclaimer to appear there, too.